website/data/menu/main.yml

```diff
@@ -974,6 +974,8 @@ main:
           - name: robinhood
             ref: "/portfolio/brokers/robinhood"
             sub:
+              - name: login
+                ref: "/portfolio/portfolio/brokers/robinhood/login"
```

Comment:
We need to remove this because this folder doesn't exist. Not sure why this didn't break in the tests...